### PR TITLE
image for wasi 22.04

### PR DESCRIPTION
**Pull request by mkhamoyan**
Contributes to https://github.com/dotnet/runtime/issues/65895

Image with

emscripten
wasi sdk
wasmtime

based on ubuntu 22.04

### CODE REVIEW

File: src/ubuntu/22.04/webassembly/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    && git clone https://github.com/emscripten-core/emsdk.git ${EMSDK_PATH} \
+    && cd ${EMSDK_PATH} \
+    && ./emsdk install ${EMSCRIPTEN_VERSION}-upstream \
+    && ./emsdk activate ${EMSCRIPTEN_VERSION}-upstream \
+    && chmod -R 777 ${EMSCRIPTEN_PATH}
+
+# Install V8 Engine
+SHELL ["/bin/bash", "-c"]
+
+ENV V8_VERSION=8.5.183
```
Comment (pavelsavara) [inline src/ubuntu/22.04/webassembly/Dockerfile:43]: Latest V8 is 11.1.218 but I don't know if we have specific preference. @radical @maraf

Comment (radical) [reply src/ubuntu/22.04/webassembly/Dockerfile:43]: Maybe the latest stable? like https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/757

Comment (mkhamoyan) [reply src/ubuntu/22.04/webassembly/Dockerfile:43]: Do I need to update also for 20.04/webassembly/Dockerfile?

**Review by mthalman — COMMENTED** [review]
Is this meant to replace the following:

https://github.com/dotnet/runtime/blob/4544bddc57c0a602c557b450ed23bd2e550438c0/eng/pipelines/common/templates/pipeline-with-resources.yml#L63-L64

Or where will it be used? If it's going to be a replacement, it would be better to also remove the 20.04 version.

File: src/ubuntu/22.04/webassembly/Dockerfile (hunk 1 above)
Comment (pavelsavara) [reply src/ubuntu/22.04/webassembly/Dockerfile]: That would be good. are there yet another images?

**Comment by mkhamoyan** [comment]
Closing this as we will not this for now

**Pull request closed without merging**